### PR TITLE
global shared changes

### PR DESCRIPTION
**Pull request by paigemiller**
change global to shared in run.py

### CODE REVIEW

File: epxresults/run.py [hunk 1]
```diff
@@ -73,9 +73,9 @@ class FREDRun(object):
     get_state :
         get state counts in a condition
     get_variable :
-        get the value of a global variable
+        get the value of a shared variable
```
Comment (duncandc) [inline epxresults/run.py:76]:
```suggestion
        get the value of a shared numeric variable
```

File: epxresults/run.py [hunk 2]
```diff
@@ -272,12 +272,12 @@ def get_state(
 
     def get_variable(self, variable: str, interval: str = "daily") -> pd.Series:
         """
-        Return an series of values for a global variable.
+        Return an series of values for a shared variable.
```
Comment (duncandc) [inline epxresults/run.py:275]:
```suggestion
        Return an series of values for a shared numeric variable.
```

File: epxresults/run.py [hunk 3]
```diff
@@ -289,7 +289,7 @@ def get_variable(self, variable: str, interval: str = "daily") -> pd.Series:
 
         Examples
         --------
-        In the ``'simpleflu'`` model, there are a set of global variables,
+        In the ``'simpleflu'`` model, there are a set of shared variables,
```
Comment (duncandc) [inline epxresults/run.py:292]:
```suggestion
        In the ``'simpleflu'`` model, there are a set of shared numeric variables,
```

File: epxresults/run.py [hunk 4]
```diff
@@ -334,12 +334,12 @@ def get_list_variable(
         sim_day: int = None,
     ) -> pd.Series:
         """
-        Return an series of values for a global list variable.
+        Return an series of values for a shared list variable.
```
Comment (tommynelson81) [inline epxresults/run.py:337]: "an" should be "a"

File: epxresults/run.py [hunk 5]
```diff
@@ -351,7 +351,7 @@ def get_list_variable(
 
         Examples
         --------
-        In the ``'simpleflu'`` model, there are a set of global list variables,
+        In the ``'simpleflu'`` model, there are a set of shared list variables,
```
Comment (tommynelson81) [inline epxresults/run.py:354]: "there are a set" should be "there is a set".

**Review by tommynelson81 — COMMENTED** [review]
Left a couple of comments outlining additional edits to text.

**Pull request merged**